src/Target.h

```diff
@@ -289,11 +289,16 @@ struct Target {
         return natural_vector_size(type_of<data_t>());
     }
 
+    /** Return true iff 64 bits and has_feature(LargeBuffers). */
+    bool use_large_buffers() const {
```

Comment:
use_large_buffers -> has_large_buffers?

use_large_buffers sounds like a directive to allocate more memory.